### ISSUE TITLE
Tutorial Bug

### ISSUE THREAD

**Issue by romanyacik** [vegastrike/Vega-Strike-Engine-Source#589]
Tutorial instructions stop being update after Oswald says to turn away from his ship and set speed to max. The tutorial mission does not end, it just does not progress. I am running Vega Strike version 0.7.0 on Linux Mint 20.2, 64 bit

**Comment by evertvorster** [comment]
You can get around this by pressing "Y" and then "\" again. Your ship will now speed up enough for Oswald to accept the speed and then you can move on to the next thing. The same happens again later when he asks you to go into reverse. 
If you have a joystick with a throttle connected and set up, you won't be able to go into reverse, because the throttle stops at 0.
Anyways, he asks for a negative speed of 20, and your ship in Maneuver mode can't make that, so press Y again to put it into travel mode and then increase the backwards speed some more.

**Comment by stephengtuggy** [comment]
@evertvorster Good to know.

**Comment by royfalk** [comment]
This may have something to do with the physics model in settings.
I remember playing the game around 2006-2010 and seeing the Llama speeds jump by a multiplier of 2 or 3.
The quest file is modules/quest_tutorial.py
Line 455 if (angle>=0.20 and velocity>=295):
So something changed at some point.

**Comment by stephengtuggy** [comment]
Yes. At one point, I believe the previous developers set the maximum speed of all ships to 300 across the board, replacing whatever individual max speeds each model had previously. That training mission / quest velocity threshold is probably a relic of that time frame.

(We later changed the ship speeds back.)
